electron/src/main.ts

```diff
@@ -544,11 +544,10 @@ class ElectronWrapperInit {
             contents.on('console-message', async (_event, _level, message) => {
               const webViewId = lifecycle.getWebViewId(contents);
               if (webViewId) {
-                const logFilePath = path.join(LOG_DIR, webViewId, config.logFileName);
                 try {
-                  await LogFactory.writeMessage(message, logFilePath);
+                  await LogFactory.writeMessage(message.replace(/%c(.+)%c /, '$1 '), LOG_FILE);
```

Comment:
Please put the regex `/%c(.+)%c` into a variable which gives information that it is about the color code.